### DISCUSSION TITLE
Combinig multiple layerStyles

### DISCUSSION THREAD

**Discussion by ParallelUniv3rse** [chakra-ui/chakra-ui#3413] [Q&A] [answered]
Is it possible to combine multiple layerStyles? 
e.g. I want to apply withOutline and withBoxShadow to a button or a card at the same time?

**Comment by dodas** [comment, marked as answer]
Hey @ParallelUniv3rse,

unfortunately, it's not currently possible.
I would recommend using a className for that. You can extend the theme and add [global styles](https://chakra-ui.com/docs/features/global-styles) to do that.
In the future we will probably bring a uniform approach that will solve these kind of problems and remove the need for `textStyles`, `layerStyles` and `apply`.
